cmd/jsutils/getchainstatus.js

```
@@ -0,0 +1,781 @@
+    } catch (error) {
+        // the "parlia" module was not enabled for RPC call
+        parliaEnabled = false;
+        console.log("parlia RPC modudle is not enabled\n", error);
+    }
+
+    for (let i = program.startNum; i < program.endNum; i++) {
+        let txCount = await provider.send("eth_getBlockTransactionCountByNumber", [ethers.toQuantity(i)]);
+        txCountTotal += ethers.toNumber(txCount);
+
+        let header = await provider.send("eth_getHeaderByNumber", [ethers.toQuantity(i)]);
+        let gasUsed = eval(eval(header.gasUsed).toString(10));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)